packages/apps/esm-implementer-tools-app/src/configuration/interactive-editor/config-subtree.component.tsx

```diff
@@ -32,25 +33,27 @@ export function ConfigSubtree({ config, path = [] }: ConfigSubtreeProps) {
 
   return (
     <>
-      {Object.entries(config).map(([key, value], i) => {
-        const thisPath = path.concat([key]);
-        const isLeaf = value.hasOwnProperty('_value') || value.hasOwnProperty('_type');
-        return (
-          <Subtree
-            label={key}
-            leaf={isLeaf}
-            onMouseEnter={() => setActiveItemDescriptionOnMouseEnter(thisPath, key, value)}
-            onMouseLeave={() => removeActiveItemDescriptionOnMouseLeave(thisPath)}
-            key={`subtree-${thisPath.join('.')}`}
-          >
-            {isLeaf ? (
-              <EditableValue path={thisPath} element={value} />
-            ) : (
-              <ConfigSubtree config={value} path={thisPath} />
-            )}
-          </Subtree>
-        );
-      })}
+      {Object.entries(config)
+        .filter(([key]) => !key.startsWith('_'))
+        .map(([key, value], i) => {
+          const thisPath = path.concat([key]);
+          const isLeaf = Object.hasOwn(value, '_value');
```

Comment:
```suggestion
          const isLeaf = value && typeof value === 'object' && Object.hasOwn(value, '_value');
```

Can we add a type check here before calling  `Object.hasOwn()`? It will throw a TypeError if called on `null` or `undefined`. Without a guard, hovering over or rendering a config node with a null/undefined value will crash.